_drafts/submitting-your-first-pull-request.markdown

```diff
@@ -0,0 +1,205 @@
+---
+layout: post
+title: Submitting Your First Pull Request
+author: dave_paquette
+---
+
+Over the last few years, we have seen a big shift in the .NET community towards open source. In addition to a huge number of open source community led projects, we have also seen Microsoft move major portions of the .NET framework over to GitHub. 
+
+With all these packages out in the wild, the opportunities to contribute are endless. The process however can be a little daunting for first timers, especially if you are not using git in your day-to-day work. In this post I will guide you through the process of submitting your first pull request. I will be showing examples from my experience contributing to the [Humanitarian Toolbox](http://www.htbox.org/)’s [allReady project](https://github.com/HTBox/allReady). As with all things git related, there is more than one way to do everything. This post will outline the workflow I have been using and should serve as a good starting point for most .NET developers who are interested in getting started with open source projects hosted on GitHub.
+
+## Installing GitHub for Windows
+
+The first step is to install [GitHub for Windows](https://desktop.github.com/). GitHub’s Windows desktop app is great, but the installer also installs the excellent posh-git command line tools. We will be using a combination of the desktop app and the command line tools.
+
+## Forking a Repo
+
+The next step is to fork the repository to which you are hoping to contribute. It is very unlikely that you will have permissions to check in code directly to actual repo. Those permissions are reserved to the project owners. The process instead is the fork the repo. 
+
+_image_
+
+This creates a branch that you own and have control over. You will be making changes in your fork, then submitting a pull request to get your changes merged in to the original repo.
+
+_image_
+
+Notice on my fork’s master branch where it says _This branch is even with HTBox:master_. _HTBox:master_ is the master branch from the original repo and is the upstream for my master branch. When GitHub tells me my branch is even with master that means no changes have happened to HTBox:master and no changes have happened to my master branch. Both branches are identical at this point.
```

Comment:
Suggestion: _This branch is even with HTBox:master_. The branch _HTBox:master_ is the...

Otherwise, it looks like the second sentence is a continuation of the first because of the continuation of the italics